v1.9/kubeadm/README.md

```diff
@@ -0,0 +1,112 @@
+# kubeadm
+Official documentation:
+ - https://kubernetes.io/docs/setup/independent/install-kubeadm/
+ - https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/
+
+
+# Reproduce Conformance Results:
+
+## Pre-Reqs
+- vagrant + virtualbox
+
+## Provision
```

Comment:
So in the future if we want to reference the https://github.com/heptio/aws-quickstart as a much easier repro-provisioning just let me know.  We are refreshing for 1.9 at the end of the month.

Reply:
I'd honestly prefer this to be as vendor-neutral as we can... => as I said I'd welcome Heptio as a "corporate backer" or "point of contact for these results as well"